docker-compose.yaml

```diff
@@ -47,7 +47,7 @@ services:
       - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
       - POSTGRES_USER=${DATABASE_USER}
     volumes:
-      - db:/var/lib/postgresql/data
+      - /data/db:/var/lib/postgresql/data
```

Comment:
hardcoding the path like this is a bad practice. Why can't we just use db volume?

Reply:
Fixed